### PR TITLE
scaletesting: update docs to reflect infrastructural changes

### PR DESCRIPTION
**Pull request by sanderginn**
- Part of https://github.com/sourcegraph/sourcegraph/issues/47886

### CODE REVIEW

**Comment by sanderginn** [comment]
Current dependencies on/for this PR:
* main
  * **PR #6256** <a href="https://app.graphite.dev/github/pr/sourcegraph/handbook/6256" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/sourcegraph/handbook/6256?utm_source=stack-comment).

File: content/departments/engineering/dev/tools/scaletesting.md [hunk 1]
```diff
@@ -18,48 +18,48 @@ The common vocabulary being used to talk about scaling from the persective of a
 
 ➡️ If you plan to run a test, announce yourself on #wg-test-at-scale to ensure you're the only one using it at the moment.
 
-It is deployed in its own Google Cloud Project and is maintained by the Developer Experience team. This is very much a collaborative effort and any help to improve it is welcomed.
+The services are deployed the Google Cloud projects `sourcegraph-scaletesting` and `sourcegraph-dogfood`, and are maintained by the Developer Experience team. This is very much a collaborative effort and any help to improve it is welcomed.
```
Comment (davejrt) [inline content/departments/engineering/dev/tools/scaletesting.md]:
```suggestion
The services are deployed in the Google Cloud projects `sourcegraph-scaletesting` and `sourcegraph-dogfood`, and are maintained by the Developer Experience team. This is very much a collaborative effort and any help to improve it is welcomed.
```

File: content/departments/engineering/dev/tools/scaletesting.md [hunk 2]
```diff
@@ -121,57 +121,51 @@ If you have already configured and authenticated to the cluster, you can of cour
...
-First, authenticate yourself to the cluser using the following command:
+First, authenticate yourself to the cluster using the following command:
 
 ```shell
-gcloud container clusters get-credentials scaletesting --region us-central1 --project sourcegraph-scaletesting
+gcloud container clusters get-credentials dogfood --region us-central1 --project sourcegraph-dogfood
 ```
 
-Then check that you have the sourcegraph helm chart installed:
+Then check that you have the Sourcegraph Gelm chart installed:
```
Comment (davejrt) [inline content/departments/engineering/dev/tools/scaletesting.md]:
```suggestion
Then check that you have the Sourcegraph Helm chart installed:
```

Comment (sanderginn) [reply content/departments/engineering/dev/tools/scaletesting.md]: GELM

File: content/departments/engineering/dev/tools/scaletesting.md [hunk 3]
```diff
@@ -18,48 +18,48 @@ The common vocabulary being used to talk about scaling from the persective of a
...
-11. Downscale the cluster to zero nodes.
 
 ### Environment
 
 The code for this environment can be found in the following locations, depending on which part of the scale testing cluster you need to change.
 
 #### Infrastructure
 
-The configuration for the Google Cloud infrastructure can be found be found in the [sourcegraph/infrastructure](https://github.com/sourcegraph/infrastructure/tree/main/scaletesting) repository. Here you will find configuration for the project, GKE cluster, Cloud SQL instances, secrets and anything else related to the configuraion of underlying components. It is not expected that these values will change often, nor should testing engineers be expected to manage this code, however contributions are always welcomed.
+The configuration for the Google Cloud infrastructure can be found be found in the [sourcegraph/infrastructure](https://sourcegraph.sourcegraph.com/github.com/sourcegraph/infrastructure@main) repository. The configuration is defined in the following directories:
```
Comment (mucles) [inline content/departments/engineering/dev/tools/scaletesting.md]:
```suggestion
The configuration for the Google Cloud infrastructure can be found in the [sourcegraph/infrastructure](https://sourcegraph.sourcegraph.com/github.com/sourcegraph/infrastructure@main) repository. The configuration is defined in the following directories:
```

File: content/departments/engineering/dev/tools/scaletesting.md [hunk 4]
```diff
@@ -106,13 +106,13 @@ The possibility of using isolated code hosts solely for the purpose of these tes
 
 In order to gather meaningful results of running tests against the scale testing instance, you can gather the following resources to help you come to a conclusion:
 
-- Tracing: Selective tracing is enabled and traces are exported to GCP. You can see all traces by going to the [GCP tracing dashboard](https://console.cloud.google.com/traces/overview?project=sourcegraph-scaletesting). To use tracing all you have to add is `&trace=1` to you url the UI will show a `View trace` link, which takes you to the GCP dashboard for your particular trace.
+- Tracing: Selective tracing is enabled and traces are exported to GCP. You can see all traces by going to the [GCP tracing dashboard](<https://console.cloud.google.com/traces/list?referrer=search&authuser=2&project=sourcegraph-dogfood&pageState=(%22traceFilter%22:(%22chips%22:%22%255B%257B_22k_22_3A_22net.host.name_22_2C_22t_22_3A10_2C_22v_22_3A_22_5C_22scaletesting.sgdev.org_5C_22_22%257D%255D%22))>). To use tracing all you have to add is `&trace=1` to you url the UI will show a `View trace` link, which takes you to the GCP dashboard for your particular trace.
```
Comment (mucles) [inline content/departments/engineering/dev/tools/scaletesting.md]:
```suggestion
- Tracing: Selective tracing is enabled and traces are exported to GCP. You can see all traces by going to the [GCP tracing dashboard](<https://console.cloud.google.com/traces/list?referrer=search&authuser=2&project=sourcegraph-dogfood&pageState=(%22traceFilter%22:(%22chips%22:%22%255B%257B_22k_22_3A_22net.host.name_22_2C_22t_22_3A10_2C_22v_22_3A_22_5C_22scaletesting.sgdev.org_5C_22_22%257D%255D%22))>). To use tracing all you have to add is `&trace=1` to the url and the UI will show a `View trace` link, which takes you to the GCP dashboard for your particular trace.
```

**Pull request merged**